backend/routes_chemicals.py

```
@@ -2,16 +2,26 @@
 from models import db, Chemical, ChemicalIssuance, User, AuditLog, UserActivity
 from datetime import datetime, timedelta
 from functools import wraps
+from utils.error_handler import handle_errors, ValidationError, log_security_event, validate_input
```

Ruff (0.11.9)

`utils.error_handler.validate_input` imported but unused

Remove unused import: `utils.error_handler.validate_input`

(F401)